docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/fractribution/index.md

```diff
@@ -72,7 +72,7 @@ vars:
     snowplow__path_transforms: {'exposure_path' : null}
 ```
 
-:::tip Snowflake Only
+
```

Comment:
You left the closing tag here if you meant to remove it at all?

Reply:
That was an accidentaly cut instead of copy